src/lib/geometry.f90

```diff
@@ -2588,6 +3145,53 @@ function coord_at_xi(ne,xi,basis)
 
   end function coord_at_xi
 
+
+!
+!###################################################################################
+!
+  subroutine splitwords(line, words, nw )
```

Comment:
Subroutine or function?

Reply:
Belongs in geometry or utilities?

Reply:
Belongs to utilities. Will clean it up after raising another issue , lots of such functions/subroutines currently in geometry